app/controllers/idv_controller.rb

```diff
@@ -28,6 +28,13 @@ def fail
 
   private
 
+  def sp_below_quota_limit?
```

Comment:
Can we call this something else? Or split it up? I don't expect predicate methods to have side effects like redirecting like this

Reply:
sure